avaframe/com1DFAPy/com1DFA.py

```diff
@@ -397,7 +397,7 @@ def initializeSimulation(cfg, demOri, releaseLine, entLine, resLine, logName, re
 
     # -----------------------
     # Initialize mesh
-    dem = initializeMesh(dem)
+    dem = initializeMesh(dem, num=8)
```

Comment:
Is the num option a development feature? If so, define a defaultNum at the top of the file and mark it. 
If we want this permanently, move it to the ini file

Reply:
num = 4 is the default
I will add it to the ini file but then we start having a lot in the ini